contracts/Makefile

```diff
@@ -5,26 +5,16 @@ all: test
 
 .PHONY: test
 test:
-	$(MAKE) foundry CMD="forge test -vvv"
+	forge test -vvv
 
 .PHONY: deploy
 deploy:
-	$(MAKE) foundry CMD="forge script script/Deploy.s.sol:Deploy --rpc-url $(RPC_URL) --private-key $(PK) --broadcast -vvv"
-
-.PHONY: foundry
-foundry:
-	docker run --rm \
-		-v $$(pwd):/src:z -w /src \
-		ghcr.io/paradigmxyz/foundry-alphanet:latest \
```

Comment:
We don't need the specialized EIP-3074 testing version anymore, so just use regular `foundry`.